apiary.apib

```diff
@@ -2239,6 +2240,653 @@ The `Errors` field in the JSON will have the following format:
 
 + Response 200 (application/json)
 
+
+### Intermediate Results [GET /measurements/{ID}/results/{chunkOrder}]
+
+This endpoint is used to get the intermediate results of a measurement done. The intermediate results are valid for 2 minutes after the measurement is done.
+
+
++ Parameters
+    + ID (string) - Requested measurement UUID.
+    + chunkOrder(number) - The intermediate result number.
+
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer <api token>
+
++ Response 200 (application/json)
+
+       {
+    "MeasurementID": "663b8ac1-67b5-4a2a-939b-6bdae2abca32",
+    "MeasurementDataID": "663b8ac1-67b5-4a2a-939b-6bdae2abca32:1",
+    "MeasurementResultID": "9f8ae87c-b899-45ce-8b5f-8e57f753cfa9",
+    "Multiplier": 10000,
+    "Channels": {
```

Comment:
Please only include `HR_BPM` and `SNR` in the results as an example.

Reply:
I don't have that Examples. Do you have one?

Reply:
It's actually in your current example, but I pasted it here again:

"SNR": {
            "Data": [
                1000000
            ],
            "Channel": "slow/toi4"
        }
"HR_BPM": {
            "Data": [
                909906
            ],
            "Channel": "slow/toi4"
        }